### PR TITLE
[15.0][MIG] stock_picking_filter_lot

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**

### CODE REVIEW

**Comment by BernatPForgeFlow** [comment]
@ForgeFlow
@AaronHForgeFlow 
@LoisRForgeFlow

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests :+1:

File: stock_picking_filter_lot/__manifest__.py [hunk 1]
```diff
@@ -1,9 +1,9 @@
-# Copyright 2018 Simone Rubino - Agile Business Group
+# Copyright 2021 Simone Rubino - Agile Business Group
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 {
     "name": "Stock picking filter lot",
     "summary": "In picking out lots' selection, filter lots based on their location",
-    "version": "14.0.1.0.1",
+    "version": "15.0.1.0.1",
```
Comment (LoisRForgeFlow) [inline stock_picking_filter_lot/__manifest__.py]: usually in migration for regular modules, the numbering is fully restarted -> 15.0.1.0.0

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**